deployment/huggingface/distilbert-sentiment-classification.ipynb

```diff
@@ -406,8 +406,7 @@
     "\n",
     "    def __init__(self, artifacts=None):\n",
     "        super(MultilingualBERT, self).__init__()\n",
-    "        self.model.return_all_scores = True  # this model has 5 categories, and we'll need to make it 2\n",
-    "\n",
+    "        self.model._postprocess_params.update({\"top_k\": None})  # this model has 5 categories, and we'll need to make it 2\n",
```

Comment:
Fixes

```python
UserWarning: `return_all_scores` is now deprecated,  if want a similar funcionality use `top_k=None` instead of `return_all_scores=True` or `top_k=1` instead of `return_all_scores=False`.
```